addons/elasticsearch-exporter/1.1.x/elasticsearch-exporter-1.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
     kubeaddons.mesosphere.io/name: elasticsearch-exporter
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-4"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-5"
```

Comment:
I have issues with this, shoudln't a new file be created for each bump?
As in `elasticsearch-exporter-2.yaml` with the bump, as we lose the previous version via this approach right?

I'd imagine for enterprise, we want users to be able to choose both versions?